process/README.md

```diff
@@ -0,0 +1,249 @@
+  - Insights
+  - EasyCLA
+  - Tech Support for DNS
+  - Crowdfunding
+  - Bevey
+- **2 Speaking slots at each OpenSSF events**
+- **Funding for Infrastructure**  
+
+#### Incubating Entry Requirements
+
+To be accepted to incubating stage, a project must meet the sandbox stage requirements plus:
+
+* Document that it is being used successfully in production by at least three independent end users which, in the TAC’s judgement, are of adequate quality and scope.
+* Have a healthy number of committers. A committer is defined as someone with the commit bit; i.e., someone who can accept contributions to some or all of the project.
+* Demonstrate a substantial ongoing flow of commits and merged contributions.
+* Since these metrics can vary significantly depending on the type, scope and size of a project, the TAC has final judgement over the level of activity that is adequate to meet these criteria
+* A clear versioning scheme.
+* Specifications must have at least one public reference implementation.
+
+#### Project Graduation Process: Incubating to Graduation
+
+Projects that wish to move from Incubating to Graduation should open a PR confirming the following criteria:
+* Have committers from at least two organizations.
+* Have achieved and maintained a [Core Infrastructure Initiative Best Practices Badge](https://bestpractices.coreinfrastructure.org/).
+* Have completed an independent and third party security audit with results published of similar scope and quality as [this example](https://github.com/envoyproxy/envoy#security-audit) which includes all critical vulnerabilities and all critical vulnerabilities need to be addressed before graduation.
+* Explicitly define a project governance and committer process. This preferably is laid out in a GOVERNANCE.md file and references an OWNERS.md file showing the current and emeritus committers.
+* Have a public list of project adopters for at least the primary repo (e.g., ADOPTERS.md or logos on the project website). For a specification, have a list of adopters for the implementation(s) of the spec.
+* Please include a short one-page narrative based off the Graduation template, no more than 500 words.
+
+#### Submission Forms for Incubating
+
+1. [Proposal](TBD)
+2. [Legal](TBD)
+
+#### Submission Forms for Incubating to Graduation
+
+1. [Proposal](TBD)
+2. [Legal](TBD)
+
+### Graduated
+
+Graduated projects signal the highest level of maturity for a OpenSSF project.
+
+#### Gives - Review and Reporting
+
+- Copyright transfer to Linux Foundation
+- Trademark transfer to Linux Foundation
+- Add Linux Foundation as GitHub Organization Owner
+- Legal Due Diligence Documentation
+- Domain Name transfer to Linux Foundation
+- Meetup Member List
+- CLA transfer to Linux Foundation
+- Reporting required by TAC and Governing Board
+
+#### Gets - Benefits
+
+- Additional governance by OpenSSF Governing Board
+- Oversight by OpenSSF TAC
+- Access to OpenSSF Ambassador program
+- Access to OpenSSF Marketing
```

Comment:
Marketing would provide Webinars, Youtube Live, Twitch, Linkedin Live, and Podcasts